src/lib/map-factory.js

```diff
@@ -1,9 +1,20 @@
+import ported from "./object-mapper/object-mapper";
+import original from "object-mapper";
 import Mapper from "./mapper";
 
-export default function createMapper() {
+export default function createMapper(options) {
+
+  let om = original;
+  let experimental = false;
+  if (!options) options = {};
+
+  if (options.experimental) {
+    om = ported;
+    experimental = true;
+  }
```

Comment:
Yes understand that... All I meant that you might not want to create variable `experimental` but to use `options.experimental` later like

```
 mapper: new Mapper(options.experimental, om)
```

But I tend to write very condensed hard-read code so it might be just me...